web/rainmaker/dev-packages/egov-estate-dev/src/ui-config/screens/specs/estate-citizen/EstateBranch_OtherCitizenService_NDC.json

```diff
@@ -3,44 +3,82 @@
   "moduleName": "EstatePropertyService",
   "EstateBranch_OtherCitizenService_NDC": [
     {
+      "dataSources": [
+        {
+          "name": "typesOfOwner",
+          "type": "local",
+          "cacheable": true,
+          "lazyload": true,
+          "values": [
+            {
+              "code": "New"
+            },
+            {
+              "code": "Existing"
+            }
+          ]
+        },
+        {
+          "name": "modesOfTransfer",
```

Comment:
modesOfTransfer is not using in this form

Reply:
changed, updated PR